src/test/java/guitests/InvalidLoginTest.java

```diff
@@ -17,9 +17,8 @@ public void launchApp() {
 
     @Test
     public void invalidLoginTest() throws InterruptedException {
-        TextField repoOwnerField = find("#repoOwnerField");
-        doubleClick(repoOwnerField);
-        doubleClick(repoOwnerField);
+        find("#repoOwnerField");
```

Comment:
I will check if the line is needed.
On Nov 8, 2015 6:01 PM, "Darius Foo" <notifications@github.com> wrote:

> In src/test/java/guitests/InvalidLoginTest.java
> <https://github.com/HubTurbo/HubTurbo/pull/1123#discussion_r44223039>:
>
> > @@ -17,9 +17,8 @@ public void launchApp() {
> >
> >      @Test
> >      public void invalidLoginTest() throws InterruptedException {
> > -        TextField repoOwnerField = find("#repoOwnerField");
> > -        doubleClick(repoOwnerField);
> > -        doubleClick(repoOwnerField);
> > +        find("#repoOwnerField");
>
> The find calls shouldn't be needed any more. Unless we want to click in
> those fields to make sure they're in focus?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/pull/1123/files#r44223039>.
>